README.rst

```diff
@@ -3,6 +3,7 @@ codespell
 
 Fix common misspellings in text files. It's designed primarily for checking
 misspelled words in source code, but it can be used with other files as well.
+It is not a general spell checker.
```

Comment:
How about something like this?
```suggestion
It does not check for word membership in a complete dictionary, but instead
looks for a set of common misspellings. Therefore it might catch errors like
"adn", but it will not catch "adnasdfasdf".
```

Reply:
Perhaps it should be "a theoretically complete" dictionary?

Reply:
Could we also make it a bit more positive, by saying "so also shouldn't generate false-positives when you use a niche term it doesn't know about".